packages/constants/typedoc.mjs

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/** @type {Partial<import("typedoc").TypeDocOptions>} */
+export default {
+  name: '@accelint/constants',
```

Comment:
i think there may be a bug in typedoc related to using the 'packages' entry point strategy. for whatever reason, context.project.name in the `createSignature` event was an empty string, unless we explicitly set the name like this in each package. in theory, it should just be getting this from package.json....

Reply:
Interesting. Not too surprising.